src/src/rx_main.cpp

```diff
@@ -790,7 +800,7 @@ static void ICACHE_RAM_ATTR ProcessRfPacket_RC(OTA_Packet_s const * const otaPkt
         return;
     }
 
-    bool telemetryConfirmValue = OtaUnpackChannelData(otaPktPtr, &crsf, ExpressLRS_currTlmDenom);
+    bool telemetryConfirmValue = OtaUnpackChannelData(otaPktPtr, CRSF::ChannelData, ExpressLRS_currTlmDenom);
```

Comment:
Is `CRSF::ChannelData` a descriptive buffer to unpack channels into, given we are now sending them to SBUS, not CRSF?

Reply:
No, but thats where they are unpacked to. I'll look at extracting that out somewhere too then.

Reply:
This one is a bigger job, but it should be done.

Reply:
Can't we just extract out the channel buffer that we currently have to a global buffer (global to rx_main), and then pass the pointer to the buffer to the CRSF class (if we are using CRSF), or any other protocol class?

Reply:
Also, I'd be ok to let this slide in this PR if its too big of a job, and we can keep it in mind as a future improvement.

Reply:
Changes to extract `ChannelData[]` look good. Thanks for pulling that out!